src/ui/src/pages/app/bootstrap_monitors/Index/components/MonitorForm/index.tsx

```diff
@@ -23,6 +23,7 @@ const MonitorForm: React.FC = () => {
   const [type, setType] = useState<any>('table_health');
   const [database, setDatabase] = useState<any>('');
   const [schema, setSchema] = useState<any>('');
+  const [daysAgo, setDaysAgo] = useState<any>(100);
```

Comment:
The type here should technically be number since the input is a number

Reply:
The type for the others should technically be string since the input is a string. Filed under technical debt not related to this change, but a good callout, and no reason not to improve the code as we write it.